### ISSUE TITLE
Dynamically generated types for js sdk

### ISSUE THREAD

**Issue by lumir-mrkva** [galacticcouncil/Basilisk-node#136]
Would be nice to have type definitions up to date all the time.

blocked by:
- [x] https://github.com/paritytech/substrate/pull/8615

**Comment by jak-pan** [comment]
This can be closed I guess?

**Comment by lumir-mrkva** [comment]
are we actually using generated types, or we are still relying on those defined in the polkadot js / readme?
